engine/src/main/scala/cromwell/engine/workflow/lifecycle/MaterializeWorkflowDescriptorActor.scala

```diff
@@ -235,9 +235,10 @@ class MaterializeWorkflowDescriptorActor(serviceRegistryActor: ActorRef,
 
     for {
       coercedInputs <- validateCoercedInputs(rawInputs, namespace)
-      _ = pushWfInputsToMetadataService(coercedInputs)
-      evaluatedWorkflowsDeclarations <- validateDeclarations(namespace, workflowOptions, coercedInputs, pathBuilders)
-      declarationsAndInputs <- checkTypes(evaluatedWorkflowsDeclarations ++ coercedInputs)
+      coercedValidatedFileInputs <- validateWdlFiles(coercedInputs)
+      _ = pushWfInputsToMetadataService(coercedValidatedFileInputs)
```

Comment:
I'm venturing into unknown territory here as I don't really know what this does, but just checking that you want to send this off somewhere before validating the rest?